### PR TITLE
Docs for cloud migration

### PR DESCRIPTION
**Pull request by disrani-px**

### CODE REVIEW

File: cloud/migration-pxctl.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Pre-requisites
+* Requires PX-Enterprise v2.0+ 
+* Make sure you have configured a [secret store](https://docs.portworx.com/secrets/) on both your clusters. This will be used to store the credentials for the 
+objectstore.
+
+## Pairing clusters
+
+1. Install portworx on two clusters
+2. On the destination cluster, run the following commands from one of the Portworx nodes (steps (a) and (b) will not be
```
Comment (talakad) [inline cloud/migration-pxctl.md]: Why not on source cluster?  May be explain a little here..

File: cloud/migration-pxctl.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+   ```
+   $ /opt/pwx/bin/pxctl cloudmigrate status 
+
+   CLUSTER UUID: 2937523c-a8f6-4564-a683-e3b53b92a3b7
+   TASK-ID                                  VOLUME-ID           VOLUME-NAME  STAGE  STATUS      LAST-UPDATE
+   107655ea-0f66-4ffe-99e2-1ef06434aa40     589129994411792979  testVolume   Done   Complete    Sat, 27 Oct 2018 01:12:40 UTC
+   ```
+4. The stages of migration will progress from Backup→ Restore→Done. If any stage fails the status will be marked as Failed.
+5. If the migration is successful you should be volume(s) with the same name created on the destination cluster.
```
Comment (talakad) [inline cloud/migration-pxctl.md]: you should be able to see/list volumes(s)

**Review by talakad — APPROVED** [review]

File: cloud/migration-pxctl.md (hunk 1 above)
Comment (disrani-px) [reply cloud/migration-pxctl.md]: Done.

File: cloud/migration-pxctl.md (hunk 2 above)
Comment (disrani-px) [reply cloud/migration-pxctl.md]: Fixed

**Pull request merged**